queue_services/entity-emailer/Makefile

```diff
@@ -54,6 +54,7 @@ install: clean ## Install python virtrual environment
 install-dev: ## Install local application
 	. venv/bin/activate ; \
 	pip install -Ur requirements/dev.txt; \
+	pip install -Ur requirements/bcregistry-libraries.txt
 	pip install -e .
```

Comment:
I'm not positive about this change.  It's not clear to me how the schemas are loaded during the CI pipeline.

Reply:
Update: If I remove the line above, the restoration tests fail locally because the restoration business schema has not been updated:
```
test_restoration_notification.py::test_complete_full_restoration_notification_includes_notice_of_articles_and_incorporation_cert FAILED [ 96%]
test_restoration_notification.py::test_paid_restoration_notification_includes_receipt_and_restoration_application_attachments FAILED [ 97%]
test_restoration_notification.py::test_completed_full_restoration_notification FAILED    [ 98%]
test_restoration_notification.py::test_completed_extended_restoration_notification FAILED [ 99%]
test_restoration_notification.py::test_paid_restoration_notification FAILED              [100%]
```

But, when I include the line above, rerun `make setup`, I get this error:
```sh
$ make test
. venv/bin/activate && pytest
ImportError while loading conftest '/home/jlonge/code/lear/queue_services/entity-emailer/tests/conftest.py'.
tests/conftest.py:25: in <module>
    from entity_emailer.config import get_named_config  # noqa: I001
E   ModuleNotFoundError: No module named 'entity_emailer'
make: *** [Makefile:72: test] Error 4

```